.profile.d/set_host.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [[ -z "${HEROKU_APP_NAME}" ]]; then
```

Comment:
This might lead to wrong host variable on production environment if the HEROKU_APP_NAME is present there. This should only be set in non production environments